templates/ORG/password-policy.yaml

```diff
@@ -11,4 +11,5 @@ Resources:
       RequireSymbols: true
       AllowUsersToChangePassword: false
       HardExpiry: false
+      PasswordReusePrevention: 24
```

Comment:
that's pretty extreme since we already require MFA.  Just wondering, is there a compliance requirement for us to be this extreme?

Reply:
Yes, please see IT-1793. This should impact no one since the design is to disallow IAM users